releasenotes/notes/fix-UCC-and-fully-occupied-MOs-c8fdf252a0607395.yaml

```diff
@@ -0,0 +1,12 @@
+---
+prelude: >
+    The UCC ansatz doesn't allow to calculate systems with either, or both of
+    the spin registers is fully occupied.
+features:
+  - |
+    Allow for building UCC ansatze for systems where at least one of the spin
+    registers is not fully occupied.
+fixes:
```

Comment:
For such a fix/change we would not have a prelude. This is combined to give an high level overview of significance for the release. I do not think in this case its warranted at all. Then I think either this a fix or a new feature (the latter meaning the UCC chose not to cater to this rather). Whatever seems, better I would have just one section rather than saying similar things around both fix and feature.

Reply:
Done :white_check_mark: